src/SqlBindingUtilities.cs

```diff
@@ -161,6 +161,99 @@ public static SqlCommand BuildCommand(SqlAttribute attribute, SqlConnection conn
             return result;
         }
 
+        /// <summary>
+        /// Returns [tableName] if tableName is not prefixed by a schema, otherwise returns [schema].[table] in the case that
+        /// tableName = schema.table
+        /// </summary>
+        /// <param name="tableName">The name of the user's table</param>
+        /// <returns>The normalized table name</returns>
+        public static string NormalizeTableName(string tableName)
```

Comment:
You should use https://github.com/Azure/azure-functions-sql-extension/blob/main/src/SqlObject.cs for parsing object names